packages/server/src/api/todo/adx/repo.ts

```diff
@@ -95,6 +94,10 @@ export default function (server: Server) {
       throw new AuthRequiredError()
     }
     const tx = input.body
+    const update = tx.writes.findIndex((write) => write.action === 'update')
+    if (update > -1) {
```

Comment:
This looks perfect, just wanted to advertise a similar option, which can be a little more intuitive without the fun `-1` business 😆 
```suggestion
    const hasUpdate = tx.writes.some((write) => write.action === 'update')
    if (hasUpdate) {
```

Reply:
oh much better 👍